pkg/pillar/types/zedroutertypes.go

```diff
@@ -1940,6 +1966,14 @@ type NetworkInstanceInfo struct {
 	// to this bridge and their statistics.
 	// We add statistics from all vifs while reporting to cloud.
 	VifMetricMap map[string]NetworkMetric
+
+	// Assigned AppNums on this NetworkInstance
+	AppNumBitMap Bitmap
```

Comment:
I suspect this will be a local field inside the appNumOnNet allocator and not part of the network instance.